plugins/source/hackernews/docs/overview.md

```diff
@@ -1,4 +1,4 @@
-The Hacker News Source plugin for CloudQuery extracts configuration from the [Hacker News API](https://github.com/HackerNews/API) and loads it into any supported CloudQuery destination (e.g. PostgreSQL, BigQuery, Snowflake, and [more](/docs/plugins/destinations/overview)).
```

Comment:
I think we want to keep the relative link here `/docs/plugins/destinations/overview`.
See https://hub.cloudquery.io/plugins/source/cloudquery/hackernews/latest/docs

Since the link is relative, when you click `more` it opens in the same tab. If you'd use the full path it will open in a new table.
Also full paths won't work in Vercel deploy previews (they'll always point to the main deploy instead of the preview deploy)